app/javascript/components/molecules/SubstatusDropdown.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { components } from 'react-select'
+import classNames from 'classnames'
+
+import {
+  LEASE_UP_STATUS_VALUES,
+  LEASE_UP_SUBSTATUS_OPTIONS,
+  LEASE_UP_SUBSTATUS_VALUES
+} from '~/utils/statusUtils'
+
+import Icon from '../atoms/Icon'
+import Dropdown from '../molecules/Dropdown'
+
+const renderSubstatusOption = ({ value, label }, { selectValue }) => {
+  const isSelected = selectValue[0]?.value === value
+  return (
+    <li className={'dropdown-menu_item'} aria-selected={isSelected}>
+      <a>{label}</a>
+    </li>
+  )
+}
+
+const SubstatusDropdown = ({
+  status,
+  subStatus,
+  onChange,
+  disabled,
+  placeholder,
+  expand,
+  hasError
+}) => {
+  const buttonClasses = [
+    'button',
+    'dropdown-button',
+    'substatus',
+    'has-icon--right',
+    'text-align-left',
+    { 'expand': expand },
+    { 'error': hasError }
+  ]
+  const renderStatusToggle = ({ children, getValue, ...props }) => {
```

Comment:
what are the other props that can be passed in here? Is it just a couple that can be named instead?

Reply:
I'm not sure, they're passed by Select and they don't necessarily enumerate them in the docs. I think there might be a lot though.